web-common/src/features/dashboards/pivot/PivotTable.svelte

```diff
@@ -12,9 +12,17 @@
   import type { Readable } from "svelte/motion";
   import { derived } from "svelte/store";
   import type { PivotDataRow, PivotDataStore } from "./types";
+  import {
+    createVirtualizer,
+    defaultRangeExtractor,
+  } from "@tanstack/svelte-virtual";
 
   export let pivotDataStore: PivotDataStore;
 
+  const OVERSCAN = 80;
```

Comment:
Overscan of 80 seems excessive? Probably can make it 30-40

Reply:
I've spent a good bit of time tinkering with this value. There is no downside to setting it at 80. Any browser is more than capable of stably rendering ~200 rows. What they can't do, within the context of TanStack's virtualization paradigm, is quickly mount/unmount/rerender huge chunks rows, which is what happens when you have a lower overscan. Virtualization, frankly, is not needed on views that have less than 2000 total cells, so this is a way of bypassing virtualization for those smaller tables. Setting it at 80 means that you get very little flashing even when scrolling quickly. This will vary from browser to browser, of course, but there's a rough example in the screenshots below. Additionally, setting it higher means that your cell widths are being derived from a large set of values, which will cause less column size snapping in the short term.

Scrolling with OVERSCAN at 30:
<img width="345" alt="Screenshot 2024-02-16 at 9 07 51 AM" src="https://github.com/rilldata/rill/assets/120223836/973a7c27-d593-4b63-bdb3-b3c7b11f085b">

Scrolling with OVERSCAN at 80:
<img width="358" alt="Screenshot 2024-02-16 at 9 07 27 AM" src="https://github.com/rilldata/rill/assets/120223836/1273cade-3094-4c50-9156-af6a57d5ec17">